pinot-query-planner/src/test/resources/queries/PinotHintablePlans.json

```diff
@@ -1,33 +1,19 @@
 {
   "pinot_hint_option_tests": {
     "queries": [
-      {
-        "description": "SELECT * inner join with filter on one table",
-        "sql": "EXPLAIN PLAN FOR SELECT /*+ joinOptions(is_colocated_by_join_keys='true') */ * FROM a JOIN b ON a.col1 = b.col2 WHERE a.col3 >= 0",
-        "output": [
-          "Execution Plan",
-          "\nLogicalJoin(condition=[=($0, $8)], joinType=[inner])",
-          "\n  PinotLogicalExchange(distribution=[single])",
-          "\n    LogicalFilter(condition=[>=($2, 0)])",
-          "\n      LogicalTableScan(table=[[a]])",
-          "\n  PinotLogicalExchange(distribution=[single])",
-          "\n    LogicalTableScan(table=[[b]])",
-          "\n"
-        ]
-      },
       {
         "description": "Inner join with group by",
-        "sql": "EXPLAIN PLAN FOR SELECT /*+ joinOptions(is_colocated_by_join_keys='true'), aggOptions(is_partitioned_by_group_by_keys='true') */a.col1, AVG(b.col3) FROM a JOIN b ON a.col1 = b.col2  WHERE a.col3 >= 0 AND a.col2 = 'a' AND b.col3 < 0 GROUP BY a.col1",
+        "sql": "EXPLAIN PLAN FOR SELECT /*+ aggOptions(is_partitioned_by_group_by_keys='true') */ a.col1, AVG(b.col3) FROM a JOIN b ON a.col1 = b.col2  WHERE a.col3 >= 0 AND a.col2 = 'a' AND b.col3 < 0 GROUP BY a.col1",
```

Comment:
The reason why I didn't add plan test for this hint is because this hint is not applied during the planning time, so the plan is always the same with/without the hint

Reply:
as @Jackie-Jiang mentioned the table hints are not altering the logical plan so the only thing we can test here is verify the hints were attached properly (e.g. modify the TableScanNode toString method)

however, since we added physical plan in https://github.com/apache/pinot/pull/11052 we can try to verify the physical plan but that's definitely a follow up IMO